lib/model/rwfolder.go

```diff
@@ -288,13 +302,20 @@ func (f *sendReceiveFolder) Serve() {
 			default:
 				l.Infoln("Completed initial scan (rw) of", f.Description())
 				close(f.initialScanCompleted)
+				if fsWatcher.WatchingFs {
+					f.delayFullScan()
```

Comment:
I wouldn't scrutinize the entire rwfolder.go part too much (apart from how an `FsWatcher` is started). There are some issue there and I had improvements in a PR together with rofolder integration, but dropped that due to incompatibility with the sendreceive PR. I will redo it once that gets finalized.